docs/setup.rst

```diff
@@ -0,0 +1,28 @@
+Account setup
+=============
+
+Creating an account on StarkNet
+-------------------------------
+
+An account is needed to start interacting with StarkNet. There are a few ways of creating one:
+
+ - using DeployAccount transaction
+ - deploy through Cairo syscall (another account is needed)
+
+The first approach is recommended since relying on third-party contracts is not necessary.
+
+Deploying an account with DeployAccount transaction requires the following:
+
+ - class_hash of the account contract
+ - generating an account's secrets
+ - computing an address based on the account's secrets
```

Comment:
Is secrets standard terminology for this? Perhaps we should just write `account private key and deployment salt` or something along the lines?

Reply:
Not sure, maybe your suggestion will be more readable